docs/Early Access - Using Global Profile Configuration.md

```diff
@@ -0,0 +1,194 @@
+
+    ```
+    zowe plugins uninstall @zowe/secure-credential-store-for-zowe-cli
+    ```
+
+You can now configure the CLI and issue commands.
+## Initializing global configuration
+
+To begin, define a connection to z/OSMF and initialize your configuration files. We recommend this method for getting started, but you can choose create the configuration files manually if desired.
+
+**Follow these steps:**
+
+1. Issue the following command:
+
+   ```
+   zowe cnfg init --global
+   ```
+
+   The CLI provides a series of prompts.
+
+2. Respond to the prompts to enter a service name, username, and password for a mainframe service such as z/OSMF. The `--global` option ensures that your credentials are stored securely on your computer by default.
+
+   After you respond to the prompts, the following two files are added to your local `.zowe/settings` directory:
+
+   - `config.json` - A *global* configuration file. This is the primary location where your MF service connection details such as host, port, etc... are defined. We recommend that you begin by working with this file.
+
+   - `config.user.json` - A *user-specific* configuration file. When you initialize configuration with the `--global` option, this file is created as an exact copy of your `config.json`. Values that you change in this file override the values that defined in `config.json`.
+
+      Alternatively, you can specify `--user` on your initialization command to generate *only* a user config file. Note that your credentials will be stored in plaintext by default when you use this option.
+
+3. Issue a Zowe CLI command to test that you can access z/OSMF. For example, list all data sets under your user ID:
+
+   ```
+   zowe zos-files list data-set "MY.DATASET.*"
+   ```
+
+   A list of data sets is returned. You successfully configured Zowe CLI to access a z/OSMF instance!
+
+   If the CLI returns an error message, verify that you have access to the target system. Examine your configuration files in a text editor to verify that the information you entered is correct.
+
+**Important!:** After the configuration files are in place (either via the `zowe cnfg init` command or by manually creating the files), the old `zowe profiles` commands will no longer function. Expect to see errors if you attempt to make use of old profiles.
+
+## Editing configuration
+
+After the initial setup, you can define additional mainframe services to your global or user config.
+
+Open the `/.zowe/settings/config.json` file in a text editor or IDE on your computer. The JSON arrays contain your initial z/OSMF connection details. For example:
+
+<!-- Insert an example here of the json for just a simple z/osmf instance, for a visual -->
+
+From here, you can edit the details as needed and save the file. For example, you might change the password field if your mainframe password changed.
```

Comment:
I would use host or port info as an example. Username and password should be specified in the `secure` array and therefore will not be edited in plain text. We could note how to update secure values though. Either `zowe config secure` to iterate through all secure properties prompting for new values or use the  `zowe config set` command to set a single secure property.